libraries/AP_Airspeed/AP_Airspeed_SST_ND.cpp

```diff
@@ -0,0 +1,292 @@
+        case DevModel::SST_ND:
+            _available_ranges = ARRAY_SIZE(vn131cm_range);
+            p_current_range = vn131cm_range;
+            _range_setting = 7;
+            break;
+        case DevModel::ND210:
+            _available_ranges = ARRAY_SIZE(nd210_range);
+            p_current_range = nd210_range;
+            _range_setting = 6;
+            break;
+        case DevModel::ND005D:
+            _available_ranges = ARRAY_SIZE(nd005d_range);
+            p_current_range = nd005d_range;
+            _range_setting = 6;
+            break;
+        case DevModel::ND160:
+            _available_ranges = ARRAY_SIZE(nd160_range);
+            p_current_range = nd160_range;
+            _range_setting = 7;
+            break;
+        case DevModel::ND130:
+            _available_ranges = ARRAY_SIZE(nd130_range);
+            p_current_range = nd130_range;
+            _range_setting = 6;
+            break;
+    }
+    
+    // Set current range value
+    _current_range_val = *(p_current_range + _range_setting);
+
+    // drop to 10 retries for runtime
+    _dev->set_retries(10);
+    
+    _dev->register_periodic_callback(20000, //  6757 for 148Hz ODR 
+                                     FUNCTOR_BIND_MEMBER(&AP_Airspeed_SST_ND::_collect, void));
+    return true;
+}
+
+bool AP_Airspeed_SST_ND::matchModel(uint8_t* model)
+{ 
+    static const struct {
+        const char *str;
+        DevModel model;
+    } models[] = {
+        { "VN131CM", DevModel::SST_ND },
+        { "ND210", DevModel::ND210 },
+        { "ND130", DevModel::ND130 },
+        { "ND160", DevModel::ND160 },
+        { "ND005D", DevModel::ND005D }
+    };
+    for (const auto &s : models) {
+        if (strcmp(s.str, (const char*) model)) {
+            continue;
+        }
+        _dev_model = s.model;
+        return true;
+    }
+    return false;
+}
+
+/*
+  convert raw temperature to temperature in degrees C
+ */
+float AP_Airspeed_SST_ND::_get_temperature(int8_t dT_int, uint8_t dT_frac) const
+{
+    return dT_int + dT_frac/256.0;
+}
+
+// read the values from the sensor
+void AP_Airspeed_SST_ND::_collect()
+{
+    uint8_t data[6]; //1 byte for mode, 3 bytes for pressure and 2 for temperature
+    WITH_SEMAPHORE(_dev->get_semaphore());
+    if (!_dev->read(data, sizeof(data))) {
+        return;
+    }
+    _dev->get_semaphore()->give();
+
+    const uint32_t dp_raw { (uint32_t)(data[1] << 16) | (data[2] << 8) | data[3] };
+
+    float press  = _get_pressure(dp_raw);
+    float temp  = _get_temperature(data[4], data[5]);
+
+    WITH_SEMAPHORE(sem);
+
+    _press_sum += press;
+    _temp_sum += temp;
+    _press_count += 1;
+    _temp_count += 1;
+
+    _last_sample_time_ms = AP_HAL::millis();
+}
+
+/*
+    convert raw pressure to pressure in Pascals
+*/
+float AP_Airspeed_SST_ND::_get_pressure(uint32_t dp_raw) const
+{
+    const float margin = 29491.2f;
+    float diff_press_inH2O = 0.0f;
+
+    if (_dev_model != DevModel::SST_ND){
```

Comment:
`switch`

.... and just nominate everything other than SST_ND in what's now the default clause.